server/rak.js

```diff
@@ -40,7 +44,12 @@ async function handleButtonPress(req, res) {
 
       if (event[0] === EVENT_TYPE.HEARTBEAT && button !== null) {
         await db.logButtonsVital(button.id, event[1], snr, rssi)
-      } else if (event[0] === EVENT_TYPE.BUTTON_PRESS_4 || event[0] === EVENT_TYPE.BUTTON_PRESS_3) {
+      } else if (
+        event[0] === EVENT_TYPE.BUTTON_PRESS_D ||
+        event[0] === EVENT_TYPE.BUTTON_PRESS_C ||
+        event[0] === EVENT_TYPE.BUTTON_PRESS_4 ||
+        event[0] === EVENT_TYPE.BUTTON_PRESS_3
```

Comment:
Change the ordering of these checks as well; 3, 4, C, D makes the most sense to me

Reply:
Done!